### PR TITLE
Track version of installed python packages

### PR DESCRIPTION
**Pull request by hikinggrass**
Get the version of the package that is to be installed and track it in the CHECK_DONE_FILE filename. This allows packages installed via the ev_create_pip_install_dist_target to be updated with newer versions without having to remove the CHECK_DONE_FILE before

### CODE REVIEW

File: python-package-helpers.cmake [hunk 1]
```diff
@@ -20,12 +20,22 @@ function (ev_create_pip_install_dist_target)
         set(EV_CREATE_PIP_INSTALL_DIST_TARGET_PACKAGE_SOURCE_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
     endif()
 
-    set(CHECK_DONE_FILE "${CMAKE_BINARY_DIR}/${EV_CREATE_PIP_INSTALL_DIST_TARGET_PACKAGE_NAME}_pip_install_dist_installed")
+    execute_process(
+        COMMAND
+            ${Python3_EXECUTABLE} -c "from setuptools import setup; setup()" --version
```
Comment (andistorm) [inline python-package-helpers.cmake:25]: Is this compatible with packages that are without `setup.py`  file?

**Review by andistorm — APPROVED** [review]
LGTM 🐧

File: python-package-helpers.cmake (hunk 1 above)
Comment (hikinggrass) [reply python-package-helpers.cmake:25]: It works with ev-cli which only uses a setup.cfg and pyproject.toml, but probably depends on your setuptools version

Comment (andistorm) [reply python-package-helpers.cmake:25]: Tested it, works also for `pyproject.toml` only ✅

**Comment by hikinggrass** [comment]
Probably not needed anymore

**Pull request closed without merging**